### PR TITLE
Create EHR Certification ID features

### PR DESCRIPTION
**Pull request by nnobrigad**
DB changes adds two new tables (ehr_certification_id and ehr_certification_product_map).

### CODE REVIEW

**Comment by andlar** [comment]
# Comments on pull requests:

## Generally:
- For every pull-request / commit, please add something to the `RELEASE_NOTES.md` file to indicate what changed. (For the data-model it'd likely be something like: "Added new tables to support CMS Certification ID"). Generally we put the new items in a "Next version" section, with a Date that's TBD, and without indicating what the new version number will actually be, as that is determined at deployment time.

## data-model:
- Both of the new tables need to have the auditing triggers added. See `dev/openchpl_audit.sql` for examples of how the auditing triggers work
- Additionally, now that the CHPL is live, all updates to the data model need to both be in the `dev/openchpl_data-model.sql` file as well as (usually) a single `.sql` file in the root directory that has commands that can be executed on a running system to update the live data model. In this case, it'd likely be a single file that has the additions you made to the data-model.sql file, just as a set of create/update lines, but that also include the auditing triggers.

**Comment by nnobrigad** [comment]
- Added two audit triggers for each ehr_certification_id and ehr_certification_id_product_map tables
- I've added the file /openchpl_ehr_certification_id.sql that contains SQL for added related DB elements

**Comment by andlar** [comment]
I'm guessing the merge conflict is due to changes in the RELEASE_NOTES file. There have been a couple of releases since the v1.1.0 that you based your work on, and the RELEASE_NOTES file has been the most common reason for a merge conflict.

Other than that, the only thing I can see is that the dev/openchpl-data_model.sql file has the triggers that the dev/openchpl_audit.sql file also has. Due to the way the various sql files grew over the development cycle, all of the triggers are concentrated in the _audit.sql file, while the table definitions are in the data-model.sql file. The sql script in the root directory has both the tables and the triggers, and that's correct, as that's the file we'll use to update a live instance. The /dev directory has the files a new instance of the system would need.

Basically, any sql files in the / directory are alterations to a live site, but those alterations also need to be in a 'from scratch' system, which is where the /dev directory comes in.

It'd probably behoove me to update the README to indicate that...

**Comment by nnobrigad** [comment]
I have removed the audit trigger statements from the dev/openchpl_data-model.sql file. So, they should only now exist within the openchpl_audit.sql and openchpl_ehr_certification_id.sql files.
